poetry/layouts/layout.py

```diff
@@ -98,6 +100,14 @@ def generate_poetry_content(self):
         if self._license:
             poetry_content["license"] = self._license
 
+        if self._readme_format not in ACCEPTED_README_FORMATS:
+            raise ValueError("Invalid readme format: {}".format(self._readme_format))
+
+        if self._readme_format == "rst":
```

Comment:
What do you think of `poetry_content["readme"] = f'README.{self._readme_format'}'`? Might be nicer since you already have a check in line 103 anyway for the accepted formats.

Reply:
Yes, that's a good idea.  But instead of f-strings, we have to you the `format()`-method, because python2 has to be supported. Correct?

Reply:
Yup, you're right.